packages/css/src/components/badge/README.md

```diff
@@ -7,6 +7,10 @@ Guides the user in taking a specific action or describes its surrounding content
 
 ## Design
 
-The badge can contain a short text or a number.
-Any colour from the ‘highlight’ and ‘feedback’ groups can be used as a background colour.
-The default is dark green.
+- The badge can contain a short text or a number.
+- Any [colour](/docs/brand-design-tokens-colour--docs) from the ‘highlight’ and ‘feedback’ groups can be used for the background.
+  The default is green.
+  The design system does not define a meaning to any of these background colours, although it makes sense to use green and red for positive and negative feedback.
```

Comment:
We do define a meaning to the colours in the 'feedback' group, though? You should not use a red badge for neutral content, I think?

Reply:
Yes, improved the guidelines on this.